SOLIDHomework.Core/Services/ShoppingCart.cs

```diff
@@ -11,23 +12,25 @@ namespace SOLIDHomework.Core
     //
     public class ShoppingCart
```

Comment:
Add abstraction for ShoppingCartService to follow Dependency Inversion principle and to work with DI container

Reply:
IinterFace for ShoppingCartService